### DISCUSSION TITLE
Is there a pure tensorflow implementation of whisper ?

### DISCUSSION THREAD

**Discussion by monowaranjum** [openai/whisper#1953] [Ideas]
Hi, I am trying to compile the model for an edge device. For the compilation task, I need the model in a tensorflow saved_model format . I tried performing the conversion from current pytorch model to tensorflow but it didn't work due to various tensorflow related issue. My question is simple: Is there a pure tensorflow implementation of whisper done by anyone which can be loaded and saved as tenorflows saved_model format ? Thanks in advance

**Comment by phineas-pta** [comment]
u may want onnx

as everyone using pytorch now, tf should be considered legacy
